api/src/main/java/org/eclipse/microprofile/lra/annotation/AfterLRA.java

```diff
@@ -37,21 +37,21 @@
  *
  * <p>
  * The listener can register interest in the final outcome of an LRA at
- * any time up until the LRA has closed or cancelled. In other words,
+ * any time up until the LRA has reached a terminal state. In other words,
  * if an LRA is closing or cancelling then listener registrations
  * should be allowed. This is in contrast to registering for participant
  * callbacks which are only allowed if the LRA is active.
  * A consequence of this statement is that if a class is annotated with
- * both the AfterLRA and the Compensate annotations and the LRA has
- * already started closing or cancelling then the method invocation
- * will fail with a <code>412 PreCondition Failed</code> JAX-RS response
- * code.
+ * both the <code>AfterLRA</code> and the {@link Compensate} annotations and the LRA has
+ * already started closing or cancelling then the {@link LRA} method invocation
+ * will fail with a <code>412 Precondition Failed</code> JAX-RS response
+ * code because of the {@link Compensate} method.
```

Comment:
AfterLRA prohibits after LRA enlistments also in Closing/Cancelling states. The enlistment fails because there is also compensate method that doesn't allow these two states for the participant enlistment. But I will try to rephrase this.

Reply:
I see, thank you for elaboration.